modules/azurerm-storage-account/variables.tf

```diff
@@ -0,0 +1,38 @@
+variable "account_replication_type" {
+  description = "Defines the type of replication to use for this storage account. Valid options are LRS, GRS, RAGRS, ZRS, GZRS and RAGZRS. Changing this forces a new resource to be created when types LRS, GRS and RAGRS are changed to ZRS, GZRS or RAGZRS and vice versa."
```

Comment:
Do we want to add a note about the default value here (and for the `account_tier`)?